configs/stage/roles/ocm.json

```diff
@@ -0,0 +1,52 @@
+{
+  "roles": [
+    {
+      "name": "OCM Cluster Editor",
+      "display_name": "Cluster Editor",
```

Comment:
Just to confirm, the `display_name` value is what will render in the UI, not the `name`. Generally these are kept the same unless the `display_name` needs to change in the UI, as `name` is the identifier for querying existing records for updates, so we're unable to modify that.